common/presenters/message-banner/move-to-message-banner-component.test.js

```diff
@@ -74,6 +79,44 @@ describe('Presenters', function () {
             })
 
             context('with assessment', function () {
+              context('with handed over assessment', function () {
+                beforeEach(function () {
+                  output = presenter({
+                    ...mockArgs,
+                    move: {
+                      ...mockMove,
+                      profile: {
+                        person_escort_record: {
+                          id: '12345',
+                          status: 'confirmed',
+                          handover_occurred_at: '2018-10-10T14:30:00Z',
+                        },
+                      },
+                    },
+                  })
+                })
+
+                it('should return unconfirmed banner', function () {
```

Comment:
Is this the unconfirmed banner, if it's calling `assignmentToHandedOverBanner`?

Reply:
Good spot. Copy/paste error.